projects/components/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Why this change? Can you please explain in the PR description?

Reply:
Sorry, forgot about this change. Adding to description and improving PR